lib/trino-filesystem/src/test/java/io/trino/filesystem/AbstractTestTrinoFileSystem.java

```diff
@@ -1221,6 +1229,111 @@ private Location createListingLocation(String path)
         return createLocation(path);
     }
 
+    @Test
+    void testFileDoesNotExistUntilClosed()
+    {
+        assertThatNoException().isThrownBy(() -> {
```

Comment:
does `assertThatNoException().isThrownBy(() -> {`  add any value in the test code?

Reply:
thanks for explaining
we don't use this pattern for tests, so please remove